docs/package/gluon-harden-dropbear.rst

```diff
@@ -0,0 +1,31 @@
+gluon-harden-dropbear
+=====================
+
+This package reduces the attack surface of dropbear, the SSH server on the router.
+
+If the root account either has no password configured or is locked,
+password-authorization is disabled in dropbear's settings.
+
+
```

Comment:
Duplicate blank line

Reply:
resolved in 4e85c03234f0ed20958b7824f4ab285ae1ca7329